### PR TITLE
Create load data

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about loading and creates score game. the following requirements have been done:
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+const update = async () => {
+  await fetch(`${URL}games/${gameData.id}/scores/`, {
+    method: 'GET',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      // console.log('url:', `${URL}games/${gameData.id}/scores/`);
+      // console.log(data);
```
Comment (rloterh) [inline src/index.js]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,68 @@
...
+const form = document.querySelector('form');
+const name = document.querySelector('input[type="text"]');
+const score = document.querySelector('input[type="number"]');
+const p = document.querySelector('.square');
+const btn = document.querySelector('.btn');
+const gameData = {
+  id: '',
+};
+
+const update = async () => {
```
Comment (rloterh) [inline src/index.js:16]: - [ ] Data is not being sent to the API correctly. When a page of the app is refreshed already entered **data vanishes**. Kindly ensure to improve upon this aspect.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Azy, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Descriptive commit message
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wandji20 — APPROVED** [review]
## Hi @azy64  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 

Your project has been approved, however, Keep up the good work.

## Congratulations :+1:  :fireworks: 

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**